### PR TITLE
Add C header generation as output file for ML4F testing. 

### PR DESCRIPTION
**Pull request by microbit-carlos**
To contain the samples used for testing, the filter output, and the model output.
This file can then be imported into a MakeCode project to compare the results from CreateAI vs the on-device implementation.

Currently still moving the test files from pxt-microbit-ml to pxt-microbit-ml-runner, so leaving this as a draft PR in case I tweak the define names.

### CODE REVIEW

**Comment by microbit-robert** [comment]
Push a commit to remove the first test file added and no longer used as well as clean up logging and exclude the ml4f-output file from testing. This should get it to pass in CI.

**Comment by microbit-carlos** [comment]
with these changes `npm run ml4f-output` doesn't work anymore, do we need to edit the vite config file whenever we want to run the test? Or is there a different method to run it?

**Comment by microbit-robert** [comment]
> with these changes `npm run ml4f-output` doesn't work anymore, do we need to edit the vite config file whenever we want to run the test? Or is there a different method to run it?

Whoops. This change should sort it [4948058](https://github.com/microbit-foundation/ml-trainer/pull/445/commits/494805899033fd0ab64c76cfc24d42148a07a07f).

File: src/makecode/ml4f-output.test.ts [hunk 1]
```diff
@@ -158,8 +175,8 @@ test("spits out files useful for testing", async () => {
   // staging/release branch current settings: samples_period = 25; samples_length = 80;
   // one second branch current settings: samples_period = 24; samples_length = 41;
   // half time branch current settings: samples_period = 24; samples_length = 38;
-  const samples_period = 24;
-  const samples_length = 41;
+  const samples_period = 20;
```
Comment (microbit-carlos) [inline src/makecode/ml4f-output.test.ts:178]: Also changed this 20, but it won't be used by the on-device testing code.

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**